src/actions/warehouseActions.js

```diff
@@ -1,9 +1,38 @@
 import ActionTypes from './actionTypes';
 import ActionUtil from './actionUtil';
+import {getRoaster} from './roasterActions';
 
 const WAREHOUSE_URL = 'https://warehouse.expresso.store/api';
 const ITEMS_URL = WAREHOUSE_URL + '/item';
 
+function receiveItem(payload) {
+    return {
+        type: ActionTypes.WAREHOUSE_ITEM,
+        payload
+    };
+}
+
 export function getAllItems(offset, limit) {
     return ActionUtil.handlePagedRequest(ActionTypes.WAREHOUSE_ITEMS, ITEMS_URL, 'GET', offset, limit);
 }
+
+export function getItem(id) {
+    return dispatch => {
```

Comment:
this could probably be generalized to something in `actionUtil` like 'getSingle` or something.